public/documents/upload_facilities.csv

```diff
@@ -1 +1 @@
-organization,facility_group,facility_name,facility_short_name,facility_type,street_address (optional),village_or_colony (optional),zone_or_block,district,pin (optional),latitude (optional),longitude (optional),size (optional),enable_diabetes_management (true/false),enable_monthly_screening_reports (true/false),enable_monthly_supplies_reports (true/false)
```

Comment:
Unrelated change?

Reply:
we're following practice to have a new line at bottom of file.